_pages/es/index.md

```diff
@@ -15,7 +15,7 @@ A campaign for Mozillians around the world to have impact in areas key to Mozill
 
 {: #area-deveng .area}
 [![image](/asserts/img/development.png)
-Empower developer communities]({{ "developer-engagement" | prepend: page.url | prepend: site.baseurl }})
+Empower developer communities](/es/developer-engagement/)
```

Comment:
I've tested this locally and without this fix, it makes http://127.0.0.1:4000/es/developer-engagement/ out of it and also shows the Spanish file content. Could you please test that again?

![screen shot 2016-08-16 at 17 59 39](https://cloud.githubusercontent.com/assets/330324/17706158/3c5d5292-63db-11e6-95c1-8ae83690e186.png)